### PR TITLE
nanopolish

### PR DESCRIPTION
**Pull request by mmiladi**

### CODE REVIEW

**Comment by mmiladi** [comment]
@bgruening variants test should not be passed. It's passed even with missing macros.xml file :-|

**Comment by bgruening** [comment]
Tests are not running ;)
You need to include the .shed.yml file :)

**Comment by mmiladi** [comment]
Now it's failing and I feel much more comfortably @bgruening ;-)

**Comment by mmiladi** [comment]
https://travis-ci.org/bgruening/galaxytools/jobs/384644047#L1616

**Comment by bgruening** [comment]
index is now passing.
The other one is failing because the output does not match: https://travis-ci.org/bgruening/galaxytools/jobs/384653325#L1985

File: tools/GraphClust/Plotting/MotifFinderPlot.py [hunk 1]
```diff
@@ -10,33 +10,74 @@
 import itertools
 import seaborn as sns
 import numpy as np
```
Comment (bgruening) [inline tools/GraphClust/Plotting/MotifFinderPlot.py]: Oo what is this?

Comment (mmiladi) [reply tools/GraphClust/Plotting/MotifFinderPlot.py]: Oo damn. `add -u` :-|

**Comment by mmiladi** [comment]
I'll open a new PR.. Sorry

**Pull request closed without merging**